tools/setup_path.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
I might make this optional in some way. It's a bit unsporting to automatically adjust folks' `PATH`, especially in a way that would result in many copies of the same directory if it's run multiple times.

Reply:
I've removed the reference of this from the `how-to-run` doc

Reply:
Oh good catch. I read that in `how-to-run.adoc` and transposed it into a script. Sorry about that!